contrib/PyCuAmpcor/README.md

```diff
@@ -0,0 +1,442 @@
+
+**Parameters**
+
+| PyCuAmpcor          | CUDA variable       | ampcor.F equivalent   | Notes                     |
+| :---                | :---                | :----                 | :---                      |
+| corrSurfaceZoomInWindow | zoomWindowSize  | i_cw   | The size of correlation surface of the (anti-aliasing) oversampled reference/secondary windows, also used to set halfZoomWindowSizeRaw. Set it to 16 to be consistent with RIOPAC. |
+
+**Difference to ROIPAC**
+
+In RIOPAC, an extra resizing step is performed on the correlation surface, from (2\*halfZoomWindowSizeRaw\*rawDataOversamplingFactor+1, 2\*halfZoomWindowSizeRaw\*rawDataOversamplingFactor+1) to (i_cw, i_cw), centered at the peak (in RIOPAC, the peak seeking is incorporated in the correlation module while is seperate in PyCuAmpcor). i_cw is a user configurable variable; it could be smaller or bigger than 2\*i_srchp\*i_ovs+1=17 (fixed), leading to extraction or enlargement by padding 0s. This procedure is not performed in PyCuAmpcor, as it makes little difference in the next oversampling procedure.
+
+### 5.6 Oversample the correlation surface and find the peak position
+
+* oversample the (real) correlation surface by a factor oversamplingFactor, or the resulting surface is of size (zoomWindowSize\*oversamplingFactor, zoomWindowSize\*oversamplingFactor) Two oversampling methods are offered, oversamplingMethod=0 (FFT, default), =1(sinc).
+* find the peak position in the oversampled correlation surface, OffsetZoomIn, in range zoomWindowSize\*oversamplingFactor.
+* calculate the final offset, from OffsetInit (which is the starting position of secondary window extraction in 2.4),
+
+   offset = (OffsetInit-halfSearchRange)+OffsetZoomIn/(oversamplingFactor\*rawDataOversamplingFactor)
+
+Note that this offset does not include the pre-defined gross offset. Users need to add them together if necessary.
+
+
+**Parameters**
+
+| PyCuAmpcor          | CUDA variable       | ampcor.F equivalent   | Notes                     |
+| :---                | :---                | :----                 | :---                      |
+| corrSurfaceOverSamplingFactor | oversamplingFactor  | i_covs   | The oversampling factor for the correlation surface |
+| corrSurfaceOverSamplingMethod | oversamplingMethod | i_sinc_fourier=i_sinc | The oversampling method 0=FFT, 1=sinc. |
+
+**Difference to ROIPAC**
+
+RIOPAC by default uses the sinc interpolator (one needs to change the FORTRAN code to use FFT). There is no differnce with the sinc interpolator, while for FFT, RIOPAC always enlarges the window to a power of 2.
+
+
+## 6. Additional Notes
+
+### 6.1 Sinc Oversampler
+
+The since oversampler/interpolator may be selected to oversample the correlation surface with *--corr-osm=1* in *cuDenseOffsets.py*, or *objOffset.corrSurfaceOverSamplingMethod=1*.
+
+The sinc interpolating formula is defined as
+
+ $$x(t) = \sum_{n=-\infty}^{\infty} x_n f( \Omega_c t-n )$$
```

Comment:
It looks like these tex equations aren't being rendered properly in the browser. What do you think about something like this? https://gist.github.com/a-rodin/fef3f543412d6e1ec5b6cf55bf197d7b

Reply:
That seems to be a common problem with markdown on github. I don't really like the idea of using an external generator, unless you strongly favor it. Or, I can use an ipynb which renders tex formula correctly on github.

Reply:
Yeah I agree it's not ideal, just thought I'd bring it up since it might be helpful to those reading it online. Ipynb could be a nice option if it's not too much trouble. I'll defer to your judgement on this.